src/set.js

```diff
@@ -0,0 +1,72 @@
+export default class Set {
+  constructor( arrayOfElements ) {
+    let _items = arrayOfElements || []
+    let _size = arrayOfElements ? arrayOfElements.length : 0
+
+    this.getItems = function() {
+      return _items
+    }
+    this.setItems = function( newItems ) {
+      _items = newItems
+      _size = _items.length
+    }
+
+    this.getSize = function() {
+      return _size
+    }
+    this.incrementSize = function() {
+      _size++
+    }
+    this.decrementSize = function() {
+      _size--
+    }
+  }
+
+  add( element ) {
+    let items = this.getItems()
+    if ( this.contains( element ) ) {
+      return false
```

Comment:
The do not specify a need to return false if un-addable. For mine I just included a success case and then just return an error if need be.